src/code/providers/provider-interface.ts

```diff
@@ -369,6 +373,8 @@ class CloudContent {
     }
     if (this.content.sharedDocumentId != null) {
       shared.sharedDocumentId = this.content.sharedDocumentId }
+    if (this.content.sharedDocumentUrl != null) {
```

Comment:
All of these night be cleaner as something like `shared.sharedDocumentUrl = this.content?.sharedDocumentUrl`.